### ISSUE TITLE
Segfault when debugging shader

### ISSUE THREAD

**Issue by PrototypePHX** [GLSL-Debugger/GLSL-Debugger#42]
When I click the cogwheel on the shader to debug it in the gui, the program segfaults. From what I could gather from the coredump output [1], this might be a problem with the mesa driver, but I'm not completely sure.

[1]: http://pastebin.com/5ymEhZ4T

**Comment by Solero93** [comment]
I found this issue while searching for solutions. Could you solve it since Aug 9, 2015? I'm curious. I only installed GLSL-Debugger to debug shaders, but it crashes each time I attempt doing it

**Comment by napcode** [comment]
No. Currently, there is noone actively working on the code. I'll gladly review and merge pull requests though.
